tests/TestCase/Pdf/Engine/WkHtmlToPdfEngineTest.php

```diff
@@ -75,6 +75,19 @@ public function testGetCommand()
         $result = $method->invokeArgs($Pdf->engine(), []);
         $expected = "wkhtmltopdf --quiet --print-media-type --orientation 'portrait' --page-size 'A4' --encoding 'UTF-8' --boolean --string 'value' --integer '42' --array 'first' 'firstValue' --array 'second' 'secondValue' - -";
         $this->assertEquals($expected, $result);
+
+        $Pdf = new CakePdf([
+            'engine' => [
+                'className' => 'CakePdf.WkHtmlToPdf',
+                'options' => [
+                    'cover' => 'cover.html',
+                    'toc' => true,
```

Comment:
From the `wkhtmltopdf`'s help the value for `toc` would be a string and you have instead only considered the case where it's value is `true`.

```
toc [TOC OPTION]...
  All options that can be specified for a page object can also be specified for
  a toc, further more the options from the TOC Options section can also be
  applied.
```

Reply:
You can now define options for the toc and cover page via an array. I couldn't come up with a better solution to pass the url for the cover. So it has to be passed via the `url` index.